stock_move_line_auto_fill/i18n/stock_pack_operation_auto_fill.pot

```diff
@@ -0,0 +1,62 @@
+# Translation of Odoo Server.
```

Comment:
:fire: this file

Reply:
Why? The file should be there. If it's outdated, then it will be updated on merge by bot.

Reply:
Ok, now exists two .pot files... one with the name of the module and this with other module name.

Reply:
Ah, OK, got it! The module was renamed indeed.